.gitignore

```diff
@@ -44,12 +45,6 @@ deps/dpdk*
 # OSX
 .DS_Store
 
-# Protobuf
```

Comment:
Do you have any particular reason to make this per-directory?

(btw, `sample_plugin/protobuf` also needs .gitignore for these auto-generated files)